_pages/index.md

```diff
@@ -19,25 +19,37 @@ image: /assets/img/login-gov-600x314.png
         <img alt="" src="{{ site.baseurl }}/assets/img/users.svg" height="90">
-        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue" markdown="1">Simple *and* secure for the public</h2>
-        <p>Because login.gov is a shared service,</p>
+        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue" markdown="1">
+          {% t index.column_1.heading %}
+        </h2>
+        <p>
+          {% t index.column_1.content.p_1 %}
+        </p>
         <ul class="list-reset teal-dots">
-          <li class="mb1">Users need fewer passwords and learn fewer interfaces.</li>
-          <li class="mb1">Security experts protect one service instead of many.</li>
-          <li class="mb1">Dedicated teams of design and security experts will continuously improve it.</li>
+          <li class="mb1">
+            {% t index.column_1.content.li_1 %}
+          </li>
+          <li class="mb1">
+            {% t index.column_1.content.li_2 %}
+          </li>
+          <li class="mb1">
+            {% t index.column_1.content.li_3 %}
+          </li>
         </ul>
       </div>
       <div class="col sm-col-4 px2 mb3">
         <img alt="" src="{{ site.baseurl }}/assets/img/partners.svg" height="90">
-        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Saving time and <br class="sm-show" />money</h2>
-        <p>login.gov handles software development, security operations, and customer support. This frees up government departments to focus on their missions while reducing costs and improving security.</p>
-        <p>A shared authentication platform frees up agency resources to deliver great services while reducing costs and improving security.</p>
+        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">
+          {% t index.column_2.heading %}
+        </h2>
+        {% t index.column_2.content.p_1 %}
       </div>
       <div class="col sm-col-4 px2 mb3">
         <img alt="" src="{{ site.baseurl }}/assets/img/built.svg" height="90">
-        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Built<br/>cooperatively</h2>
-        <p>login.gov works hand-in-hand with the private sector and nonprofits to identify and implement best practices and new standards in secure, privacy-protecting authentication.</p>
-        <p>Led by the <a href="https://www.usds.gov" target="_blank">U.S. Digital Service</a> and <a href="https://18f.gsa.gov" target="_blank">18F</a>, login.gov builds on groundwork laid by the <a href="http://www.nist.gov/" target="_blank">National Institute for Standards in Technology</a>, the <a href="https://www.whitehouse.gov/the-press-office/2016/02/09/fact-sheet-cybersecurity-national-action-plan" target="_blank">Cybersecurity National Action Plan</a>, and the <a href="http://www.gsa.gov/portal/content/105080" target="_blank">Federal Acquisition Service</a>.</p>
+        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">
+          {% t index.column_3.heading %}
```

Comment:
Putting so much html in these seems a bit odd, but I'm not sure how to get rid of this without placing the text in a capture tag and markdownifying it, like below. Both seem pretty clunky though. Any thoughts?
```
{% capture column3_heading %}{% t index.column_3.heading %}{% endcapture %}
{{ column3_heading | markdownify }}
```